kernel/src/fs/ramfs/fs.rs

```diff
@@ -96,7 +96,7 @@ impl FileSystem for RamFs {
 }
 
 /// An inode of `RamFs`.
-struct RamInode {
+pub struct RamInode {
```

Comment:
So is there a reason why we want to define `Memfd`-related (especially `MemfdInode`) stuffs under `vm/` instead of under `fs/`?